app/models/payroll/claims_csv.rb

```diff
@@ -0,0 +1,63 @@
+require "csv"
+
+module Payroll
+  class ClaimsCsv
+    attr_accessor :claims
+
+    FIELDS_WITH_HEADERS = {
+      title: "TITLE",
+      first_name: "FORENAME",
+      middle_name: "FORENAME2",
+      surname: "SURNAME",
+      national_insurance_number: "SS_NO",
+      payroll_gender: "GENDER",
+      start_date: "START_DATE",
+      end_date: "END_DATE",
+      date_of_birth: "BIRTH_DATE",
+      email_address: "EMAIL",
+      address_line_1: "ADDR_LINE_1",
+      address_line_2: "ADDR_LINE_2",
+      address_line_3: "ADDR_LINE_3",
+      address_line_4: "ADDR_LINE_4",
+      county: "ADDR_LINE_5",
```

Comment:
Although their sample file has a "County" under `ADDR_LINE_5`, I don't think we should interpret that to mean that the 5 line should be the county and the 6th the postcode. I read the sample as the address should appear in the address lines in the order they appear, i.e. a three line address with the postcode as the third line would have postcode under `ADDR_LINE_3`